### ISSUE TITLE
"length" validation should add "required" attribute automatically

### ISSUE THREAD

**Issue by kv109** [amatsuda/html5_validators#18]
Hi!

I've noticed that

```ruby
validates :first_name, length: { minimum: 2, maximum: 50 }
```
translates to
```html
<input maxlength="50" minlength="2" size="50" type="text">
```

<br>
To be 100% accurate with actual model validation, it should translate to
```html
<input required="required" maxlength="50" minlength="2" size="50" type="text">
```

If you agree, I could prepare a pull request with that, are you interested?

**Comment by amatsuda** [comment]
@kv109 Ah, yes. I agree. Are you still willing to work on a PR?

**Comment by kv109** [comment]
Not really. I wanted you to merge my another [PR](https://github.com/amatsuda/html5_validators/pull/20) first (you just did). But I have something different. On my own fork I did [simple handling](https://github.com/oxfordvp/html5_validators/commit/04bfcbae55a89df825e1b72fd62e1a782a138cfd) of `:if` and `:unless` options. Code is messy, since it was written just for my team, but I could rewrite it. It was helpful for us since we have a lot of validations with simple `:if` and `:unless` conditions. Let me know if you would be interested.

**Comment by Ana06** [comment]
what is the state of this? :thinking:

**Comment by EdSF** [comment]
I haven't installed this gem and just reading through issues before I do. 

Unfortunately have to **disagree** on this...it's a narrow/opinionated use case. If you want **both** `required` and `string` length validation, they should be separate (separate/distinct use cases).
